test/weights.jl

```diff
@@ -470,17 +467,17 @@ end
     @test sum([1.0, 2.0, 3.0], wt) ≈ 6.0
     @test mean([1.0, 2.0, 3.0], wt) ≈ 2.0
 
-    @test sum(a, wt, 1) ≈ sum(a, dims=1)
-    @test sum(a, wt, 2) ≈ sum(a, dims=2)
-    @test sum(a, wt, 3) ≈ sum(a, dims=3)
+    @test sum(a, wt, dims = 1) ≈ sum(a, dims = 1)
```

Comment:
We don't generally use spaces around `=` for keyword arguments AFAIK. I think the cases above are actually the exception.